### ISSUE TITLE
setup.pl "copy db" breaks both the original and copy db

### ISSUE THREAD

**Issue by sbts** [ledgersmb/LedgerSMB#3219]
LedgerSMB 1.5.12

Unfortunately when copying a db, role_prefix got set to `'lsmb___'` in the original db and not set at all in the new db

Ideally, the original DB shouldn't be altered, and the new DB should have it correctly set to
`'lsmb_originalDBname__'`

The original DB can be fixed by running something like......
```bash
psql --password --user postgres --host=$POSTGRES_HOST -c $'DELETE FROM defaults WHERE setting_key = \'role_prefix\'' $DBNAME
```

While the copy db can be fixed by
```bash
psql --password --user postgres --host=$POSTGRES_HOST -c "INSERT into defaults (setting_key, value) select 'role_prefix', '$oldDBNAME';"
```

On a side note, When this is fixed, some help text (maybe a tooltip) needs to be added to explain that copying a DB links the DB roles, hence it's not an independent copy.
This may or may-not matter in the real world, but it probably does mean it's not a good idea to use it to create multiple db's from an empty but configured template db

Specifically, the copy uses the original's roles.
This means that for a copied DB a user with the same role name has the same rights in all copies (and the original).

**Comment by ehuelsmann** [comment]
Are you sure there wasn't a role prefix already?

**Comment by sbts** [comment]
Admittedly I hadn't examined the cluster beforehand.
However it was a clean cluster with only the original db extant.
The original db was freshly created via setup.pl with the same version of lsmb

**Comment by ehuelsmann** [comment]
Fixed in #3262
